src/main/resources/db/changelog/v001-add-diagnosis-key-entity-indexes.yml

```diff
@@ -0,0 +1,19 @@
+databaseChangeLog:
+  - changeSet:
+      id: create-diagnosis-key-entity-table-indexes
+      author: dfischer-tech
+      changes:
+        - createIndex:
+            tableName: diagnosiskey
+            indexName: idx_diagnosiskey_uploader_information_batch_tag
+            columns:
+              - column:
+                  name: uploader_information_batch_tag
+                  type: varchar(64)
```

Comment:
As per [liquibase documentation](https://docs.liquibase.com/change-types/community/create-index.html)  the type seems to be redundant.

```suggestion
```